requirements.txt

```diff
@@ -1,35 +1,35 @@
-APScheduler==3.6.1
+APScheduler==3.6.3
 autobahn==20.7.1
-beautifulsoup4==4.8.0
-colorama==0.4.1
+beautifulsoup4==4.9.1
+colorama==0.4.3
 cssmin==0.2.0
 Flask==1.1.2
 Flask-Assets==2.0
-Flask-RESTful==0.3.7
+Flask-RESTful==0.3.8
 Flask-Scrypt==0.1.3.6
 httplib2==0.18.1
-irc==17.1
+irc==19.0.0
 isodate==0.6.0
 jsmin==2.2.2
-Markdown==3.1.1
-Pillow==7.1.0
+Markdown==3.2.2
+Pillow==7.2.0
 psycopg2==2.8.5
-pylast==3.1.0
+pylast==3.3.0
 pyOpenSSL==19.1.0 # for twisted.internet.ssl
 pyScss==1.3.7
 python-Levenshtein==0.12.0
-pytz==2019.2
+pytz==2020.1
 ratelimiter==1.2.0.post0
-redis==3.3.8
-regex==2019.8.19
-requests==2.22.0
+redis==3.5.3
+regex==2020.7.14
+requests==2.24.0
 retry==0.9.2
-riotwatcher==2.7.1
```

Comment:
https://github.com/pseudonym117/Riot-Watcher#changelog

⛔ 
they renamed `RiotWatcher` to `LolWatcher` (we use the old name). https://github.com/pajbot/pajbot/blob/ae4f76d84fba8f8e32f20643db82b5972c3f8409/pajbot/modules/leaguerank.py#L96
also says "removed long-deprecated classes" - we might be using them without knowing. After renaming the above class and upgrading the dependency the module should be tested